docs/docs-content/clusters/data-center/nutanix/nutanix.md

```diff
@@ -0,0 +1,53 @@
+
+The section below describes the overall workflow to register Nutanix with Palette and deploy a cluster using Palette. 
+
+## Workflow
+
+The following summary steps outline the CAPI cloud workflow illustrated in the diagram below. A system administrator performs the first step at the system level to make the cloud provider available to all the tenants. CAPI users perform the remaining steps in Palette and their local environment.
+
+![Diagram showing the overall CAPI workflow for users to construct self-hosted cloud solutions tailored to their environment.](/clusters_data-center_nutanix_workflow.png)
+
+1. A system administrator registers Nutanix with Palette by invoking system-level APIs. This includes defining the logo, required cloud account information and key-value pairs to create the Nutanix cloud account. If you do not have a cloud logo, Palette provides a default logo. 
+
+2. Create a self-hosted PCG by preparing YAML configuration templates and installing Palette agents for the PCG in your Kubernetes cluster using the kubectl commands provided in the Palette UI.
+
+  When the PCG status displays Running state, the Kubernetes cluster is integrated with Palette.
+
+  :::info
+
+  All clouds require the deployment of a PCG, which enables Palette to securely monitor clusters in the cloud. The PCG relies on the self-hosted Kubernetes cluster to remain operational.
+
+  :::
+
+3. Next, add the cloud account to Palette. The required fields to add the cloud are pre-defined by the system administrator during cloud registration in step 1. The PCG name provided at the time of PCG creation is auto-filled, and you provide the account name and account details.
+
+4. Create a cluster profile by selecting Nutanix as the cloud type. 
+
+  Palette displays out-of-the-box packs for the network and storage profile layers. Additionally, Palette provides a Nutanix-specific storage pack. If you have custom packs, you can add them to Palette by adding your registry. To learn how to add a pack registry, review [Add a Custom Registry](https://docs.spectrocloud.com/registries-and-packs/adding-a-custom-registry/#configure-a-custom-pack-registry-in-palette).
+
+5. Deploy a cluster by specifying Nutanix, listed under **Tech Preview**, as the cluster type. Select the cloud account you added and make any needed changes to the profile layers by using the YAML editor. At the **Cluster Config** step, adjust YAML configuration files to customize the cluster for your environment. You provide your own Operating System (OS) image.
+
```

Comment:
Please replace all the macros in the Cluster configuration and Node configuration pages. Macros will be mentiobed under ${}

Reply:
you might not need to replace the ones which are already specified in the cloud account.

Reply:
applied